ktor-client/ktor-client-apache5/jvm/test/io/ktor/client/engine/apache5/ApacheResponseConsumerTest.kt

```
@@ -0,0 +1,147 @@
+        val callback = object : FutureCallback<Unit> {
+            override fun completed(result: Unit) {
+                terminalCallCount.incrementAndGet()
+                firstTerminal.complete(Unit)
+            }
+
+            override fun failed(ex: Exception) {
+                terminalCallCount.incrementAndGet()
+                firstTerminal.complete(Unit)
+            }
+
+            override fun cancelled() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)